mindsdb/integrations/handlers/databricks_handler/databricks_handler.py

```diff
@@ -277,10 +325,23 @@ def get_columns(self, table_name: str, schema_name: str | None = None) -> Respon
         if not table_name or not isinstance(table_name, str):
             raise ValueError("Invalid table name provided.")
 
+        try:
+            _validate_identifier(table_name)
+            table_literal = _escape_literal(table_name)

```

Comment:
there are always `_validate_identifier` before `_escape_literal`, so may be add `_validate_identifier` to the top of `_escape_literal`